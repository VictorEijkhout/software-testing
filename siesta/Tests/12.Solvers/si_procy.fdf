%include ../base_si8.fdf
SystemLabel     si_procy

Diag.ProcessorY 2

