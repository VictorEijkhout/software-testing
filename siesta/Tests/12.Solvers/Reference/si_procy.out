Siesta Version  : 5.0.0-alpha-123-gfb8fd7d02
Architecture    : ----
Compiler version: GNU-11.4.0
Compiler flags  : -fallow-argument-mismatch;-O3 -march=native
PP flags        : ----
Libraries       : ----
Parallelisations: none
GEMM3M support
NetCDF support
NetCDF-4 support
Lua support

Runtime information:
* Directory : /home/fnpedron/Siesta/siesta-dev/Tests/12.Solvers/work/si_procy
* Running in serial mode.
>> Start of run:  28-SEP-2023  15:39:13

                           *********************** 
                           *  WELCOME TO SIESTA  * 
                           *********************** 

reinit: Reading from standard input
reinit: Dumping input in INPUT_TMP.46720
************************** Dump of input data file ****************************
%include ../../basejob.fdf
SystemLabel     si_procy
Diag.ProcessorY 2
************************** End of input data file *****************************

reinit: -----------------------------------------------------------------------
reinit: System Name: Silicon
reinit: -----------------------------------------------------------------------
reinit: System Label: si_procy
reinit: -----------------------------------------------------------------------

initatom: Reading input for the pseudopotentials and atomic orbitals ----------
Species number:   1 Atomic number:   14 Label: Si


---- Processing specs for species: Si
Ground state valence configuration:   3s02  3p02
Reading pseudopotential information in formatted form from:
  /home/fnpedron/Siesta/siesta-dev/Tests/Pseudos/Si.psf

---- Pseudopotential check for Si

Pseudized shells:
3s( 2.00) rc: 1.89
3p( 2.00) rc: 1.89
3d( 0.00) rc: 1.89
4f( 0.00) rc: 1.89
Valence configuration for ps generation: (assumed as above)
For Si, standard SIESTA heuristics set lmxkb to 3
 (one more than the basis l, including polarization orbitals).
Use PS.lmax or PS.KBprojectors blocks to override.

<basis_specs>
===============================================================================
Si                   Z=  14    Mass=  28.090        Charge= 0.17977+309
Lmxo=2 Lmxkb= 3    BasisType=split      Semic=F
L=0  Nsemic=0  Cnfigmx=3
          i=1  nzeta=1  polorb=0  (3s)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000    
            lambdas:    1.0000    
L=1  Nsemic=0  Cnfigmx=3
          i=1  nzeta=1  polorb=1  (3p)  (to be polarized perturbatively)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000    
            lambdas:    1.0000    
L=2  Nsemic=0  Cnfigmx=3
          i=1  nzeta=0  polorb=0  (3d)  (perturbative polarization orbital) (from 3p)
-------------------------------------------------------------------------------
L=0  Nkbl=1  erefs: 0.17977+309
L=1  Nkbl=1  erefs: 0.17977+309
L=2  Nkbl=1  erefs: 0.17977+309
L=3  Nkbl=1  erefs: 0.17977+309
===============================================================================
</basis_specs>

Using PAO.SplitTailNorm
atom: Called for Si                    (Z =  14)
Maximum radius (at nrval) set to   50.00000 8518

read_vps: Pseudopotential generation method:
read_vps: ATM3      Troullier-Martins                       
Valence charge in pseudo generation:    4.00000

xc_check: Exchange-correlation functional:
xc_check: Ceperley-Alder
V l=0 = -2*Zval/r beyond r=  2.5804
V l=1 = -2*Zval/r beyond r=  2.5804
V l=2 = -2*Zval/r beyond r=  2.5804
V l=3 = -2*Zval/r beyond r=  2.5804
All V_l potentials equal beyond r=  1.8729
This should be close to max(r_c) in ps generation
All pots = -2*Zval/r beyond r=  2.5804
Using large-core scheme (fit) for Vlocal

atom: Estimated core radius    2.58045

atom: Including non-local core corrections could be a good idea
Fit of Vlocal with continuous 2nd derivative
Fitting vlocal at       1.8786
Choosing vlocal chloc cutoff:***  2.821566
qtot up to nchloc:    4.00020356
atom: Maximum radius for chlocal:    2.82157
atom: Maximum radius for r*vlocal+2*Zval:    2.82157

KBgen: Kleinman-Bylander projectors: 
GHOST: No ghost state for L =  0
   l= 0   rc=  1.872931   el= -0.796617   Ekb=  5.353935   kbcos=  0.317881
GHOST: No ghost state for L =  1
   l= 1   rc=  1.886158   el= -0.307040   Ekb=  2.204864   kbcos=  0.299917
GHOST: No ghost state for L =  2
   l= 2   rc=  1.909049   el=  0.013097   Ekb= -2.473633   kbcos= -0.044292
GHOST: No ghost state for L =  3
   l= 3   rc=  1.951736   el=  0.019538   Ekb= -1.407727   kbcos= -0.001862

KBgen: Total number of Kleinman-Bylander projectors:  16
atom: -------------------------------------------------------------------------

atom: SANKEY-TYPE ORBITALS:

SPLIT: Orbitals with angular momentum L= 0

SPLIT: Basis orbitals for state 3s

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.010000 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    5.413980
                 energy =   -0.785062
                kinetic =    0.554059
    potential(screened) =   -1.339121
       potential(ionic) =   -3.805086

SPLIT: Orbitals with angular momentum L= 1

SPLIT: Basis orbitals for state 3p

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.010000 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    6.830288
                 energy =   -0.295476
                kinetic =    0.850831
    potential(screened) =   -1.146306
       potential(ionic) =   -3.389842

POLgen: Perturbative polarization orbital with L=  2

POLgen: Polarization orbital for state 3p
Split based on tail norm

   izeta = 1
                     rc =    6.830288
                 energy =    0.328538
                kinetic =    1.535909
    potential(screened) =   -1.207371
       potential(ionic) =   -3.334649
atom: Total number of Sankey-type orbitals:  9

atm_pop: Valence configuration (for local Pseudopot. screening):
 3s( 2.00)                                                            
 3p( 2.00)                                                            
 3d( 0.00)                                                            
Vna: chval, zval:    4.00000   4.00000

Vna:  Cut-off radius for the neutral-atom potential:   6.830288

atom: _________________________________________________________________________

prinput: Basis input 

PAO.BasisType split     

%block ChemicalSpeciesLabel
    1   14 Si                      # Species index, atomic number, species label
%endblock ChemicalSpeciesLabel

%block PAO.Basis                 # Define Basis set
# WARNING: This information might be incomplete!
Si                    2                    # Species label, number of l-shells
 n=3   0   1                         # n, l, Nzeta 
   5.414   
   1.000   
 n=3   1   1 P   1                   # n, l, Nzeta, Polarization, NzetaPol
   6.830   
   1.000   
%endblock PAO.Basis

prinput: ----------------------------------------------------------------------

Dumping basis to NetCDF file Si.ion.nc
coor:   Atomic-coordinates input format  =     Fractional

siesta: Atomic coordinates (Bohr) and species
siesta:      0.00000   0.00000   0.00000  1        1
siesta:      2.56530   2.56530   2.56530  1        2
siesta:      5.13061   5.13061   0.00000  1        3
siesta:      7.69591   7.69591   2.56530  1        4
siesta:      5.13061   0.00000   5.13061  1        5
siesta:      0.00000   5.13061   5.13061  1        6
siesta:      7.69591   2.56530   7.69591  1        7
siesta:      2.56530   7.69591   7.69591  1        8

siesta: System type = bulk      

initatomlists: Number of atoms, orbitals, and projectors:      8    72   128

siesta: ******************** Simulation parameters ****************************
siesta:
siesta: The following are some of the parameters of the simulation.
siesta: A complete list of the parameters used, including default values,
siesta: can be found in file out.fdf
siesta:
redata: Spin configuration                          = none
redata: Number of spin components                   = 1
redata: Time-Reversal Symmetry                      = T
redata: Spin spiral                                 = F
redata: Long output                                 =   F
redata: Number of Atomic Species                    =        1
redata: Charge density info will appear in .RHO file
redata: Write Mulliken Pop.                         = NO
redata: Matel table size (NRTAB)                    =     1024
redata: Mesh Cutoff                                 =   300.0000 Ry
redata: Net charge of the system                    =     0.0000 |e|
redata: Min. number of SCF Iter                     =        0
redata: Max. number of SCF Iter                     =     1000
redata: SCF convergence failure will abort job
redata: SCF mix quantity                            = Hamiltonian
redata: Mix DM or H after convergence               =   F
redata: Recompute H after scf cycle                 =   F
redata: Mix DM in first SCF step                    =   T
redata: Write Pulay info on disk                    =   F
redata: New DM Occupancy tolerance                  = 0.000000000001
redata: No kicks to SCF
redata: DM Mixing Weight for Kicks                  =     0.5000
redata: Require Harris convergence for SCF          =   F
redata: Harris energy tolerance for SCF             =     0.000100 eV
redata: Require DM convergence for SCF              =   T
redata: DM tolerance for SCF                        =     0.000100
redata: Require EDM convergence for SCF             =   F
redata: EDM tolerance for SCF                       =     0.001000 eV
redata: Require H convergence for SCF               =   T
redata: Hamiltonian tolerance for SCF               =     0.001000 eV
redata: Require (free) Energy convergence for SCF   =   F
redata: (free) Energy tolerance for SCF             =     0.000100 eV
redata: Using DFT-D3 dispersion                     =   F
redata: Using Saved Data (generic)                  =   F
redata: Use continuation files for DM               =   F
redata: Neglect nonoverlap interactions             =   F
redata: Method of Calculation                       = Diagonalization
redata: Electronic Temperature                      =   299.9863 K
redata: Fix the spin of the system                  =   F
redata: Max. number of TDED Iter                    =        1
redata: Number of TDED substeps                     =        3
redata: Dynamics option                             = Single-point calculation
mix.SCF: Pulay mixing                            = Pulay
mix.SCF:    Variant                              = stable
mix.SCF:    History steps                        = 2
mix.SCF:    Linear mixing weight                 =     0.250000
mix.SCF:    Mixing weight                        =     0.250000
mix.SCF:    SVD condition                        = 0.1000E-07
redata: Save all siesta data in one NC              =   F
redata: ***********************************************************************

%block SCF.Mixers
  Pulay
%endblock SCF.Mixers

%block SCF.Mixer.Pulay
  # Mixing method
  method pulay
  variant stable

  # Mixing options
  weight 0.2500
  weight.linear 0.2500
  history 2
%endblock SCF.Mixer.Pulay

DM_history_depth set to one: no extrapolation allowed by default for geometry relaxation
Size of DM history Fstack: 1
Total number of electrons:    32.000000
Total ionic charge:    32.000000

siesta: k-grid: Number of k-points = 1
siesta: k-point is Gamma-only

diag: Algorithm                                     = D&C
diag: Used triangular part                          = Lower
diag: Absolute tolerance                            =  0.100E-15
diag: Orthogonalization factor                      =  0.100E-05
diag: Memory factor                                 =  1.0000


ts: **************************************************************
ts: Save H and S matrices                           =    F
ts: Save DM and EDM matrices                        =    F
ts: Only save the overlap matrix S                  =    F
ts: **************************************************************

************************ Begin: TS CHECKS AND WARNINGS ************************
************************ End: TS CHECKS AND WARNINGS **************************


                     ====================================
                        Single-point calculation
                     ====================================

outcell: Unit cell vectors (Ang):
        5.430000    0.000000    0.000000
        0.000000    5.430000    0.000000
        0.000000    0.000000    5.430000

outcell: Cell vector modules (Ang)   :    5.430000    5.430000    5.430000
outcell: Cell angles (23,13,12) (deg):     90.0000     90.0000     90.0000
outcell: Cell volume (Ang**3)        :    160.1030
Gamma-point calculation with interaction between periodic images
Some features might not work optimally:
e.g. DM initialization from atomic data
<dSpData1D:S at geom step 0
  <sparsity:sparsity for geom step 0
    nrows_g=72 nrows=72 sparsity=1.0000 nnzs=5184, refcount: 7>
  <dData1D:(new from dSpData1D) n=5184, refcount: 1>
refcount: 1>
new_DM -- step:     1
Initializing Density Matrix...
DM filled with atomic data:
<dSpData2D:DM initialized from atoms
  <sparsity:sparsity for geom step 0
    nrows_g=72 nrows=72 sparsity=1.0000 nnzs=5184, refcount: 8>
  <dData2D:DM n=5184 m=1, refcount: 1>
refcount: 1>
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:       8     249

InitMesh: MESH = 60 x 60 x 60 = 216000
InitMesh: Mesh cutoff (required, used) =   300.000   337.446 Ry
New grid distribution [1]: sub = 2

stepf: Fermi-Dirac step function

siesta: Program's energy decomposition (eV):
siesta: Ebs     =      -301.165090
siesta: Eions   =      1967.745613
siesta: Ena     =       863.813383
siesta: Ekin    =       336.603642
siesta: Enl     =       154.780109
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =        21.601414
siesta: DUscf   =         1.894075
siesta: DUext   =         0.000000
siesta: Ex      =      -219.730389
siesta: Ec      =       -40.695969
siesta: Exc     =      -260.426358
siesta: EbV     =         0.000000
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =      -835.972974
siesta: Etot    =      -849.479349
siesta: FreeEng =      -849.479355

        iscf     Eharris(eV)        E_KS(eV)     FreeEng(eV)     dDmax    Ef(eV) dHmax(eV)
   scf:    1     -835.972974     -849.479349     -849.479355  1.255498 -4.515492  0.164562
timer: Routine,Calls,Time,% = IterSCF        1       0.683  29.21
   scf:    2     -849.484498     -849.481943     -849.481953  0.000625 -4.481777  0.119553
   scf:    3     -849.487333     -849.484779     -849.484818  0.001673 -4.390618  0.001459
   scf:    4     -849.484782     -849.484781     -849.484819  0.000039 -4.390397  0.001087
   scf:    5     -849.484786     -849.484783     -849.484821  0.000113 -4.389747  0.000033
   scf:    6     -849.484783     -849.484783     -849.484821  0.000000 -4.389741  0.000024

SCF Convergence by DM+H criterion
max |DM_out - DM_in|         :     0.0000001684
max |H_out - H_in|      (eV) :     0.0000237380
SCF cycle converged after 6 iterations

Using DM_out to compute the final energy and forces
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:       8     249

siesta: E_KS(eV) =             -849.4848

siesta: E_KS - E_eggbox =      -849.4848

siesta: Atomic forces (eV/Ang):
----------------------------------------
   Tot   -0.000000   -0.000000   -0.000000
----------------------------------------
   Max    0.000000
   Res    0.000000    sqrt( Sum f_i^2 / 3N )
----------------------------------------
   Max    0.000000    constrained

Stress tensor Voigt[x,y,z,yz,xz,xy] (kbar):      -98.89      -98.89      -98.89        0.00        0.00        0.00
(Free)E + p*V (eV/cell)     -839.6027
Target enthalpy (eV/cell)     -849.4848

siesta: Program's energy decomposition (eV):
siesta: Ebs     =      -301.563704
siesta: Eions   =      1967.745613
siesta: Ena     =       863.813383
siesta: Ekin    =       336.070453
siesta: Enl     =       154.497677
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =        22.288449
siesta: DUscf   =         1.844619
siesta: DUext   =         0.000000
siesta: Ex      =      -219.569105
siesta: Ec      =       -40.684647
siesta: Exc     =      -260.253752
siesta: EbV     =         0.000000
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =      -849.484783
siesta: Etot    =      -849.484783
siesta: FreeEng =      -849.484821

siesta: Final energy (eV):
siesta:  Band Struct. =    -301.563704
siesta:       Kinetic =     336.070453
siesta:       Hartree =      58.223608
siesta:         Edftu =       0.000000
siesta:       Eso     =       0.000000
siesta:    Ext. field =       0.000000
siesta:         Exch. =    -219.569105
siesta:         Corr. =     -40.684647
siesta:     Bulk bias =       0.000000
siesta:   Exch.-corr. =    -260.253752
siesta:  Ion-electron =    -414.736558
siesta:       Ion-ion =    -568.788534
siesta:       Ekinion =       0.000000
siesta: D3 dispersion =       0.000000
siesta:         Total =    -849.484783
siesta:         Fermi =      -4.389741

siesta: Stress tensor (static) (eV/Ang**3):
siesta:    -0.061724   -0.000000    0.000000
siesta:     0.000000   -0.061724    0.000000
siesta:     0.000000    0.000000   -0.061724

siesta: Cell volume =        160.103007 Ang**3

siesta: Pressure (static):
siesta:                Solid            Molecule  Units
siesta:           0.00067225          0.00067225  Ry/Bohr**3
siesta:           0.06172359          0.06172359  eV/Ang**3
siesta:          98.89209387         98.89209383  kBar
(Free)E+ p_basis*V_orbitals  =        -844.550677
(Free)Eharris+ p_basis*V_orbitals  =        -844.550677

cite: Please see "si_procy.bib" for an exhaustive BiBTeX file.
cite: Please clearly indicate Siesta version in published work: 5.0.0-alpha-123-gfb8fd7d02
cite: This calculation has made use of the following articles
cite: which are encouraged to be cited in a published work.
        Primary SIESTA paper
          DOI: www.doi.org/10.1088/0953-8984/14/11/302


timer: Elapsed wall time (sec) =       9.162
timer: CPU execution times (sec):

Routine            Calls   Time/call    Tot.time        %
siesta                 1       9.161       9.161   100.00
Setup                  1       0.035       0.035     0.38
bands                  1       0.000       0.000     0.00
KSV_init               1       0.000       0.000     0.00
IterGeom               1       9.125       9.125    99.61
geom_init              1       1.620       1.620    17.69
state_init             1       0.040       0.040     0.43
hsparse                1       0.002       0.002     0.03
overlap                1       0.037       0.037     0.40
Setup_H0               1       0.894       0.894     9.76
naefs                  2       0.000       0.000     0.00
MolMec                 2       0.000       0.000     0.00
kinefsm                2       0.024       0.048     0.53
nlefsm                 2       0.118       0.237     2.59
DHSCF_Init             1       0.699       0.699     7.63
DHSCF1                 1       0.019       0.019     0.21
INITMESH               1       0.000       0.000     0.00
DHSCF2                 1       0.680       0.680     7.42
REMESH                 1       0.031       0.031     0.33
REORD                 24       0.000       0.005     0.05
PHION                  1       0.591       0.591     6.45
COMM_BSC              35       0.000       0.005     0.06
POISON                 9       0.005       0.047     0.52
fft                   18       0.002       0.033     0.36
setup_H                7       0.682       4.772    52.09
DHSCF                  8       1.010       8.081    88.21
DHSCF3                 8       0.682       5.453    59.52
rhoofd                 8       0.398       3.188    34.79
XC                     8       0.016       0.128     1.40
GXC-CellXC             8       0.015       0.123     1.35
gridxc@cellXC          8       0.015       0.123     1.35
vmat                   8       0.260       2.078    22.68
IterSCF                6       0.682       4.091    44.66
compute_dm             6       0.001       0.004     0.04
diagon                 6       0.001       0.004     0.04
r-eigvec               6       0.000       0.003     0.03
r-buildHS              6       0.000       0.000     0.00
rdiag                  6       0.000       0.003     0.03
rdiag1                 6       0.000       0.000     0.00
rdiag2                 6       0.000       0.000     0.00
rdiag3                 6       0.000       0.002     0.02
rdiag4                 6       0.000       0.000     0.00
r-buildD               6       0.000       0.001     0.01
MIXER                  5       0.000       0.000     0.00
PostSCF                1       3.414       3.414    37.26
DHSCF4                 1       2.628       2.628    28.69
dfscf                  1       2.563       2.563    27.97
overfsm                1       0.013       0.013     0.14
state_analysis         1       0.000       0.000     0.00
siesta_move            1       0.000       0.000     0.00
Analysis               1       0.000       0.000     0.00
optical                1       0.000       0.000     0.00
  

>> End of run:  28-SEP-2023  15:39:22
Job completed
